Testing/SystemTests/tests/analysis/VesuvioCommandsTest.py

```diff
@@ -59,6 +59,16 @@ def runTest(self):
         runs = "15039-15045"
         self._fit_results = fit_tof(runs, flags)
 
+    def skipTests(self):
+        # Only run tests on Windows and RHEL7
+        dist = platform.linux_distribution()
+        is_rhel7 =  dist[0] == 'Red Hat Enterprise Linux Workstation' and dist[1] == '7.2'
```

Comment:
We don't want to cut out CentOS 7 (the free rebuild of Red Hat) and I think the version check is a bit to strict. How about something like
```
is_rhel7 = (('Red Hat' in dist[0]) or ('CentOS' in dist[0])) and (dist[1].startswith('7'))
```
It might be an idea to extract this whole thing to a separate function at module level that each `def skipTests(self)` can call too.